document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/mapping/mappers/exemptions/ApiToExemptionsMapper.java

```diff
@@ -27,33 +27,41 @@ void checkActiveExemption(ExemptionsApi exemptionsApi, @MappingTarget Exemptions
         if(exemptionsApi.getDisclosureTransparencyRulesChapterFiveApplies() != null) {
              disclosureTransparencyRulesChapterFiveAppliesList = exemptionsApi.getDisclosureTransparencyRulesChapterFiveApplies().getItems();
 
-            exemptions.setActiveExemption(false);
+             if(!exemptions.isActiveExemption()) {
+                 exemptions.setActiveExemption(false);
```

Comment:
Why are we setting `isActiveExemption` to `false` after checking if it is `false`?